emperor/support_files/js/view-controller.js

```diff
@@ -155,24 +155,6 @@ define([
   };
 
   /**
-   * Converts the current instance into a JSON string.
```

Comment:
Can you re-add this block, the base-class should still show that the method is not implemented, and that it is up to the subclasses to implement it.